public/api/conf/1.0/examples/example_messages/Trade Test 5.1 Sample Messages/IVL_Sample_TC01/IVL_Sample_TC01_Goods_Arrival_Message_v1.0.xml

```diff
@@ -1,5 +1,5 @@
 POST /customs/inventory-linking-imports/arrival-notifications/ HTTP/1.1
```

Comment:
how about `<INSERT_HOST_HERE>` rather than `<INSERTHOSTHERE>
` ?

Reply:
Looks better... Ive applied this to the files